Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -383,21 +359,199 @@ protected async Task OnSaveCommand()
         {
             editState = DataGridEditState.None;
 
-            if ( EditMode == DataGridEditMode.Popup )
-                PopupVisible = false;
+            return Task.CompletedTask;
         }
 
-        protected internal short ResolveItemIndex( TItem item )
+        /// <summary>
+        /// Selects the specified item.
+        /// </summary>
+        /// <param name="item">Item to select.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task Select( TItem item )
+        {
+            return SelectRow( item );
+        }
+
+        /// <summary>
+        /// Sorts the Data for the specified column.
+        /// 
+        /// Note that <see cref="DataGridColumn{TItem}.Sortable"/> must be enabled to be able to sort!
+        /// </summary>
+        /// <param name="column">Column to sort.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task Sort( DataGridColumn<TItem> column )
+        {
+            if ( Sortable && column.Sortable )
+            {
+                if ( SortMode == DataGridSortMode.Single )
+                {
+                    // in single-mode we need to reset all other columns to default state
+                    foreach ( var c in Columns.Where( x => x.Field != column.Field ) )
+                    {
+                        c.CurrentDirection = SortDirection.None;
+                    }
+
+                    // and also remove any column sort info except for current one
+                    SortByColumns.RemoveAll( x => x.Field != column.Field );
+                }
+
+                column.CurrentDirection = column.CurrentDirection.NextDirection();
+
+                if ( !ManualReadMode )
+                {
+                    if ( !SortByColumns.Any( c => c.Field == column.Field ) )
+                    {
+                        SortByColumns.Add( column );
+                    }
+                    else if ( column.CurrentDirection == SortDirection.None )
+                        SortByColumns.Remove( column );
+                }
+
+                dirtyFilter = dirtyView = true;
+
+                if ( ManualReadMode )
+                    return HandleReadData( CancellationToken.None );
+            }
+
+            return Task.CompletedTask;
+        }
+
+        /// <summary>
+        /// Triggers the DataGrid to change data source page.
+        /// </summary>
+        /// <remarks>
+        /// Valid <paramref name="pageName"/> values are:
+        /// 1-n:    Number of the page.
+        /// prev:   Go to first page.
+        /// next:   Go to next page.
+        /// first:  Go to first page.
+        /// last:   Go to last page.
+        /// </remarks>
+        /// <param name="pageName">Name or number(1 indexed) of the page.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task Paginate( string pageName )

```

Comment:
In addition to the already explanatory method doc, a more explanatory parameter name:
`string pageName` => `string pageNameOrNumber`  or  `string paginationCommandOrNumber`
something along those lines, in my opinion. Sometimes good parameter names are really helpful. :D

Reply:
Done